sentry-opentelemetry/sentry-opentelemetry-extra/src/main/java/io/sentry/opentelemetry/OtelSpanWrapper.java

```diff
@@ -129,7 +129,11 @@ public OtelSpanWrapper(
       return NoOpSpan.getInstance();
```

Comment:
this should also use makeCurrent(), even if there is no practical change, right?

Reply:
Yeah `NoOpSpan.makeCurrent` returns a `NoOpScopesLifecycleToken` atm.

Reply:
I just checked, there's more places where we would have to add this `.makeCurrent` for noop spans. I'd say we only add it when we actually need it. That's if we ever change `NoOpSpan.makeCurrent` to modify `Context`.